src/python/pants/notes/1.30.x.rst

```diff
@@ -5,6 +5,150 @@ This document describes releases leading up to the ``1.30.x`` ``stable`` series.
+  `PR #10090 <https://github.com/pantsbuild/pants/pull/10090>`_
+
+* Inline _wrap_type in `parser.py` (#10096)
+  `PR #10096 <https://github.com/pantsbuild/pants/pull/10096>`_
+
+* Make the `roots` goal display roots configured via marker files (#10079)
+  `PR #10079 <https://github.com/pantsbuild/pants/pull/10079>`_
+
+* Refactor v2 coverage and add an integration test (#10073)
+  `PR #10073 <https://github.com/pantsbuild/pants/pull/10073>`_
+
+* Extract member_type and type_arg in `parser.py` (#10063)
+  `PR #10063 <https://github.com/pantsbuild/pants/pull/10063>`_
+
+* Replace `TargetAdaptor` implementation of `list` with Target API implementation (#9912)
+  `PR #9912 <https://github.com/pantsbuild/pants/pull/9912>`_
+
+* Rename `InteractiveProcessRequest` to `InteractiveProcess` (#10050)
+  `PR #10050 <https://github.com/pantsbuild/pants/pull/10050>`_
+
+* Remove stale TODO (#10048)
+  `PR #10048 <https://github.com/pantsbuild/pants/pull/10048>`_
+
+* Reinitialize pantsd for most configuration changes (#10035)
+  `PR #10035 <https://github.com/pantsbuild/pants/pull/10035>`_
+
+* Remove unused FmtRequest union. (#9934)
+  `PR #9934 <https://github.com/pantsbuild/pants/pull/9934>`_
+
+* Use cargo-metadata not cargo to get packages (#10045)
+  `PR #10045 <https://github.com/pantsbuild/pants/pull/10045>`_
+
+* Expose new StreamingWorkunitContext on streaming workunits callback (#10034)
+  `PR #10034 <https://github.com/pantsbuild/pants/pull/10034>`_
+
+* Redesign v2 coverage mechanism (#10100)
+  `PR #10100 <https://github.com/pantsbuild/pants/pull/10100>`_
+
+Testing
+~~~~~~~
+
+* Increase timeout on test targets (#10097)
+  `PR #10097 <https://github.com/pantsbuild/pants/pull/10097>`_
+
+* Increase timeout for node_test_integration test target (#10094)
+  `PR #10094 <https://github.com/pantsbuild/pants/pull/10094>`_
+
+* Increase timeout for incomplete_custom_scala test target (#10089)
+  `PR #10089 <https://github.com/pantsbuild/pants/pull/10089>`_
+
+* remoting: add workunits for upload and execution (#10084)
+  `PR #10084 <https://github.com/pantsbuild/pants/pull/10084>`_
+
+* disable append-only cache in remoting (#10077)
+  `PR #10077 <https://github.com/pantsbuild/pants/pull/10077>`_
+
+* Push down process timeouts in remote execution (#10044)
+  `PR #10044 <https://github.com/pantsbuild/pants/pull/10044>`_
+
+* Add a __run.sh script to reproduce a process execution in a preserved chroot (#10030)
+  `PR #10030 <https://github.com/pantsbuild/pants/pull/10030>`_
+
+* Fix nightly build. (#10047)
+  `PR #10047 <https://github.com/pantsbuild/pants/pull/10047>`_
+
+Documentation
+~~~~~~~~~~~~~
+* Prepare the 1.28.1rc0 release.
+
+* Prepare the 1.27.1rc0 release.
+
+* Prepare the 1.26.1rc0 release.
+
+* Prepare the 1.25.1rc1 release.
+
+* Prepare the 1.24.1rc1 release.
+
+* Prepare the 1.23.1rc0 release.
+
+* Prepare the 1.22.1rc0 release.
+
+* Prepare the 1.21.1rc0 release.
+
+* Prepare 1.29.0. (#10074)
```

Comment:
Hm, maybe rename to `Prepare 1.21.1rc0 to 1.28.1rc0 and 1.29.0`. I think it might look weird that these are all separate list entries using the same PR url.

Reply:
adjusted to:
```
 Documentation
 ~~~~~~~~~~~~~
 * Prepare the 1.28.1rc0 release.
+  `Manual commit <https://github.com/pantsbuild/pants/commit/48b0ab940289ae7d440105224d93c9496fd59825>`_
 
 * Prepare the 1.27.1rc0 release.
+  `PR #10066 <https://github.com/pantsbuild/pants/pull/10066>`_
 
 * Prepare the 1.26.1rc0 release.
+  `PR #10067 <https://github.com/pantsbuild/pants/pull/10067>`_
 
 * Prepare the 1.25.1rc1 release.
+  `PR #10068 <https://github.com/pantsbuild/pants/pull/10068>`_
 
 * Prepare the 1.24.1rc1 release.
+  `PR #10069 <https://github.com/pantsbuild/pants/pull/10069>`_
 
 * Prepare the 1.23.1rc0 release.
+  `PR #10070 <https://github.com/pantsbuild/pants/pull/10070>`_
 
 * Prepare the 1.22.1rc0 release.
+  `PR #10071 <https://github.com/pantsbuild/pants/pull/10071>`_
 
 * Prepare the 1.21.1rc0 release.
+  `PR #10072 <https://github.com/pantsbuild/pants/pull/10072>`_
 
 * Prepare 1.29.0. (#10074)
   `PR #10074 <https://github.com/pantsbuild/pants/pull/10074>`_
```